### ISSUE TITLE
R1Q15 separate data set metrics

### ISSUE THREAD

**Issue by dagewa** [dagewa/DIALS_for_ED_paper#27]
> What were the metrics of each data set separately – completeness, resolution, etc. how were the datasets merged – any scaling factor? What is the R (any) of the experimental data compared to the theoretical intensities?

**Comment by dagewa** [comment]
This could be added, e.g. to the supplementary material, however it was not our intention here to fully reproduce the comprehensive processing already done for these datasets, as reported in [Clabbers _et. al_ 2017](http://scripts.iucr.org/cgi-bin/paper?S2059798317010348). Here we rather wanted to demonstrate that DIALS is able to produce data of comparable quality to XDS when used for electron diffraction processing.

**Comment by dagewa** [comment]
TODO: add a table to supplementary material giving data quality metrics for individual datasets

**Comment by dagewa** [comment]
Regarding the question about R, do you think the reviewer means something other than R1?

**Comment by dagewa** [comment]
About merging and scale factors - I don't think any change is required in the manuscript. It is stated clearly in section 2.9 that AIMLESS was used, and the appropriate citation is given

**Comment by dagewa** [comment]
Reply:
We have added Table S1 to the supplementary material that summarises merging statistics for each of the example datasets treated independently. Scaling and merging was done as described. We modified the text to make it clear that the independent processing jobs were used only to determine resolution cutoffs for the individual unscaled datasets, which were then scaled and merged together in a single job. The scaling model used by AIMLESS fits a physical model that is smoothed in rotation angle and resolution. The details of this procedure is outside the scope of this paper, but the appropriate citation is provided. The R factor between experimental and theoretical data is given in Table 2 (i.e. R1). Here we do not go into as much detail as the previous publication, Clabbers _et al._ (2017), in which, for example, the degree of dynamic scattering is assessed by analysis of the Fc vs Fo plot. It is not our intention to fully reproduce the comprehensive processing already done for these datasets. Rather, here we want to demonstrate that DIALS is able to produce data of comparable quality to XDS when used for electron diffraction processing.
